core/isolate.rs

```diff
@@ -315,6 +315,26 @@ impl Isolate {
       PinnedBuf::new(zero_copy_buf),
     );
 
+    let op = match op {
+      Op::Async(fut) => {
+        let mut spawn_handle = futures::sync::oneshot::spawn(
+          fut,
+          &tokio_executor::DefaultExecutor::current(),
+        );
```

Comment:
Spawning future on the executor solves problem with task tracking however... Performance is bad.
```
$ cargo run --release --example deno_core_http_bench
single-thread
http_bench.js start
listening http://127.0.0.1:4544/ rid = 3
```

```
$ third_party/wrk/mac/wrk -d 10s --latency http://127.0.0.1:4544
Running 10s test @ http://127.0.0.1:4544
  2 threads and 10 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency   144.94us   49.46us   2.89ms   95.13%
    Req/Sec    33.62k     1.97k   35.15k    93.07%
  Latency Distribution
     50%  138.00us
     75%  141.00us
     90%  151.00us
     99%  322.00us
  675670 requests in 10.10s, 32.86MB read
Requests/sec:  66899.91
Transfer/sec:      3.25MB
```